Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -3316,6 +3333,21 @@ bool CStaticFunctionDefinitions::SetHelicopterRotorSpeed(CClientVehicle& Vehicle
     return false;
 }
 
+bool CStaticFunctionDefinitions::SetVehicleRotorSpeed(CClientVehicle& Vehicle, float fSpeed)
+{
+    // It's a plane or heli?
+
+    if (Vehicle.GetVehicleType() == CLIENTVEHICLE_HELI)
+    {
+        Vehicle.SetHeliRotorSpeed(fSpeed);
+        return true;
+    }
+    else if (Vehicle.GetVehicleType() == CLIENTVEHICLE_PLANE) {
+        Vehicle.SetPlaneRotorSpeed(fSpeed);
+        return true;
+    }
```

Comment:
Can this be refactored to a `switch` statement? I think it'd look more elegant that way.

Reply:
> Can this be refactored to a `switch` statement? I think it'd look more elegant that way.

Sure, I'll do the refactoring.